Update npm release notes (#1022)
guardian/support-dotcom-components: packages/dotcom/README.md

A library for use by dotcom (frontend/DCR). This is for sharing logic and types with dotcom (frontend + dcr).

### Publishing to NPM

Releasing to NPM is handled with [changesets] and is performed by CI.

On your feature branch, before merging, run `yarn dotcom changeset` from the root of the project. This will
interactively ask you what kind of change this is (major, minor, patch) and
allow you to describe the change. Commit the generated changeset file to git and
push to your branch.

When you merge the branch, a version release PR will be automatically opened.

*IMPORTANT* - you now need push a commit to this branch which manually deletes the

> changeset file under `packages/dotcom/.changeset/`. This file is supposed to be deleted
> automatically, but it isn't, probably because of our yarn workspaces setup.

Reviewer: Nitpick: there _could_ be multiple files if we're bundling several changes together into one release.

```suggestion
changeset file(s) under `packages/dotcom/.changeset/`. These files are supposed to be deleted
automatically, but aren't, probably because of our yarn workspaces setup.
```
